tests/unit/test_backup.py

```
@@ -0,0 +1,471 @@
+    return BackupManager()
+
+
+@pytest.fixture
+def test_backup_info():
+    """Create a sample BackupInfo for testing."""
+    return BackupInfo(
+        success=True,
+        type="directory",
+        host_id="test.example.com",
+        source_path="/opt/appdata/test-stack",
+        backup_path="/tmp/docker_mcp_backups/backup_test-stack_20250101_120000.tar.gz",
```

Ruff (0.14.4)

Probable insecure usage of temporary file or directory: "/tmp/docker_mcp_backups/backup_test-stack_20250101_120000.tar.gz"

(S108)